compose/snippets/src/main/java/com/example/compose/snippets/animations/sharedelement/BasicSharedElementSnippets.kt

```diff
@@ -441,3 +443,57 @@ private object UniqueKeySnippet {
     }
     // [END android_compose_shared_elements_unique_key]
 }
+
+// [START android_compose_shared_element_scope]
+val LocalNavAnimatedVisibilityScope = compositionLocalOf<AnimatedVisibilityScope?> { null }
+val LocalSharedTransitionScope = compositionLocalOf<SharedTransitionScope?> { null }
+
+@Composable
+private fun SharedElementScope_CompositionLocal() {
+    // An example of how to use composition locals to pass around the shared transition scope, far down your UI tree.
+    // [START_EXCLUDE]
```

Comment:
Since youre giving the entire method here does it make sense to exclude these parts? If a developer copy/pastes this in they may be confused about where state should be defined

Reply:
The snippet is in context of teaching about CompositionLocal usage with SharedTransitionLayout + AnimatedContent, the expectataion is the the developer would know what state they want to provide to their AnimatedContent composable.